client/src/templates/Challenges/components/Side-Panel.js

```diff
@@ -35,7 +33,8 @@ const propTypes = {
 };
 
 export class SidePanel extends Component {
-  componentDidMount() {
+  async componentDidMount() {
+    const MathJax = await global.MathJax;
```

Comment:
```suggestion
    const MathJax = global.MathJax;
```
`MathJax` is just an ordinary object, no `await`ing required.